scene/gui/color_picker.cpp

```diff
@@ -84,10 +84,11 @@ void ColorPicker::set_pick_color(const Color &p_color) {
 	if (!is_inside_tree())
 		return;
 
+	_update_color();
+
 	return; //it crashes, so returning
```

Comment:
What's the meaning of this return? It should be removed, and if the code following it is meant not be run, it should be removed too.